thirdparty/runtime_environments.go

```diff
@@ -456,6 +516,10 @@ func (c *RuntimeEnvironmentsClient) GetImageInfo(ctx context.Context, imageID st
 		return nil, errors.Wrapf(err, "getting latest AMI and timestamp")
 	}
 	if len(amiHistory) != 1 {
+		grip.Error(message.WrapError(err, message.Fields{
+			"message":  "expected exactly 1 history result for image",
```

Comment:
Wrapping error here will cause this to noop, since error is nil, you can just error on message.Fields directly